instrumentation/opentelemetry-instrumentation-aiohttp-client/src/opentelemetry/instrumentation/aiohttp_client/__init__.py

```diff
@@ -280,7 +280,7 @@ def instrumented_init(wrapped, instance, args, kwargs):
         if context_api.get_value(_SUPPRESS_INSTRUMENTATION_KEY):
             return wrapped(*args, **kwargs)
 
-        client_trace_configs = list(kwargs.get("trace_configs", ()))
+        client_trace_configs = list(kwargs["trace_configs"] or [])
```

Comment:
If trace_configs did not exist in kwargs the code throws a KeyError exception, so you need to verify this and use kwargs.get().

client_trace_configs = list(kwargs.get("trace_configs") or [])

Reply:
Done!